roles/edpm_ovn/tasks/cleanup.yml

```diff
@@ -43,6 +43,20 @@
     - not edpm_enable_chassis_extport | bool
     - cleanup_ovn_cms_options.rc == 0
 
+- name: Cleanup availability-zones
+  block:
+    - name: Cleanup availability-zones from stdout when undefined
+      ansible.builtin.set_fact:
+        filtered_azs:
+          stdout: "{{ cleanup_ovn_cms_options.stdout | regex_replace('availability-zones=[^\"]*' ~ ',?', '') }}"
```

Comment:
AFAIU this has an assumption that may fire back later, namely that the zones is the last entry in the options list (hence "read till double-quote" instruction in the regexp.)

My first preference would be to make it not make this assumption; my second preference would be to add a comment here documenting this assumption (perhaps also in the `configure` playbook, so that whenever we touch the latter, we are aware that some other piece of code relies on the order.)

My final concern is that cleanup is not validated in tests, so if there's some problem with the regex used, we won't notice it until production deployments.

Reply:
Perhaps...?

```python
>>> x
'availability-zones=az1:az2,blabla=xxx'
>>> re.sub('availability-zones=[^\"]*,?', '', x)
''
>>> re.sub('availability-zones=[^,]*,?', '', x)
'blabla=xxx'
```

Reply:
Done, my bad somehow local changes got missed for this